docker/docker-compose.yml

```diff
@@ -516,6 +533,12 @@ services:
     networks:
       - magistrala-base-net
     volumes:
+      - ./ssl/certs/ca.key:/etc/ssl/certs/ca.key
+      - ./ssl/certs/ca.crt:/etc/ssl/certs/ca.crt
+      - ./ssl/certs/magistrala-server.crt:/etc/ssl/certs/magistrala-server.crt
+      - ./ssl/certs/magistrala-server.key:/etc/ssl/certs/magistrala-server.key
+      - ./ssl/certs/thing.crt:/etc/ssl/certs/thing.crt
+      - ./ssl/certs/thing.key:/etc/ssl/certs/thing.key
       # Things gRPC mTLS client certificates
```

Comment:
We load env cert path to const path, because in .env the actual location of certificates will be given and here we will load the const path with conditionally.
If the path given is not given , a dummy path will be loaded. but the env varabile of contaienr is not set.

```suggestion
      # MQTT Adapter certificates
      - type: bind
        source: ${MG_MQTT_ADAPTER_CERT_FILE:-./ssl/certs/dummy/server_cert}
        target: /magistrala-mqtt-adapter${MG_MQTT_ADAPTER_CERT_FILE:+.crt}
        read_only: true
        bind:
          create_host_path: true
      - type: bind
        source: ${MG_MQTT_ADAPTER_KEY_FILE:-./ssl/certs/dummy/server_key}
        target: /magistrala-mqtt-adapter${MG_MQTT_ADAPTER_KEY_FILE:+.key}
        read_only: true
        bind:
          create_host_path: true
      - type: bind
        source: ${MG_MQTT_ADAPTER_SERVER_CA_FILE:-./ssl/certs/dummy/server_ca}
        target: /magistrala-ca${MG_MQTT_ADAPTER_SERVER_CA_FILE:+.crt}
        read_only: true
        bind:
          create_host_path: true
      - type: bind
        source: ${MG_MQTT_ADAPTER_CLIENT_CA_FILE:-./ssl/certs/dummy/client_ca}
        target: /magistrala-client-ca${MG_MQTT_ADAPTER_CLIENT_CA_FILE:+.key}
        read_only: true
        bind:
          create_host_path: true
```